src/main/java/org/opentripplanner/routing/vertextype/SampleVertex.java

```diff
@@ -0,0 +1,50 @@
+package org.opentripplanner.routing.vertextype;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.opentripplanner.analyst.core.Sample;
+import org.opentripplanner.routing.edgetype.TemporaryEdge;
+import org.opentripplanner.routing.graph.Edge;
+import org.opentripplanner.routing.graph.Graph;
+
+import com.vividsolutions.jts.geom.Coordinate;
+
+/** A vertex representing a sample */
```

Comment:
Needs clarification about what a Sample is. We should avoid this kind of tautological Javadoc. Of course I know it can be distracting to write docs while working on code and this is probably just a placeholder, I'm just commenting it for revision after the merge.